app/models/announcement.rb

```diff
@@ -1,2 +1,21 @@
+# frozen_string_literal: true
+
+require 'active_support/all'
+
 class Announcement < ApplicationRecord
+  def self.populate_from_airtable
```

Comment:
i'm concerned there is no error checking.  what if the Airtable call fails? what if any of the `create` calls in line 17 fails? the user will have no way of knowing whether something happened or not, and if not, why not.

Reply:
Done.  Also replicated fixes in Task airtable fetching!